src/transform_data.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+# Author: Taiwo Owoseni
+# date: 2021-11-23
+
+"""Cleans and transforms csv file and outputs cleaned data to file path as csv file.
+
+
+Usage: src/transform_data.py --input_file=<input_file> --out_file=<out_file> --test_file=<test_file>
+
+Options:
+--input_file=<input_file>  Path (including filename) of the data to be transformed (script supports only csv)
+--out_file=<out_file>      Path (including filename) of where to locally write the file
+--test_file=<test_file>    Path (including filename) of where to locally write the test file
+"""
+
+
+import numpy as np
+import os
+import pandas as pd
+import subprocess
+from docopt import docopt
+from sklearn.compose import  make_column_transformer
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import OneHotEncoder,StandardScaler
+
+opt = docopt(__doc__)
+
+def main(input_file, output_file, test_file):
+
+    subprocess.call("src/download.py --out_type=xls --url=https://archive.ics.uci.edu/ml/machine-learning-databases/00350/default%20of%20credit%20card%20clients.xls --out_file=data/raw/default_credit.csv", shell=True)
```

Comment:
I think we need oto delete this part, and just assume that the data is there and read. @thayeylolu

Reply:
Thanks @liannah. The reason it is there is because I initially interpreted this `A second script that reads the data from the first script` to mean running the subprocess to execute the first script

Reply:
Thank you for changing. The issue is resolved. I don’t have any other comments as of now.

Reply:
Also please make sure to add doc string to each function

Reply:
Forgot to include Docstring!

Reply:
The first issue is resolved.